### PR TITLE
feat(profiling): add view select menu

### PR DESCRIPTION
**Pull request by JonasBa**
CleanShot 2022-02-09 at 09 23 15

@Zylphrex we may need some smaller buttons 🙈

### CODE REVIEW

File: static/app/components/profiling/FlamegraphViewSelectMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  onSortingChange,
+}: FlamegraphViewSelectMenuProps): React.ReactElement {
+  return (
+    <ButtonBar merged>
+      <Button
+        size="xsmall"
+        priority={sorting === 'call order' ? 'primary' : undefined}
+        onClick={() => onSortingChange('call order')}
+      >
+        Call Order
```
Comment (Zylphrex) [inline static/app/components/profiling/FlamegraphViewSelectMenu.tsx:24]:
```suggestion
        t('Call Order')
```

File: static/app/components/profiling/FlamegraphViewSelectMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      </Button>
+      <Button
+        size="xsmall"
+        priority={sorting === 'left heavy' ? 'primary' : undefined}
+        onClick={() => onSortingChange('left heavy')}
+      >
+        Left Heavy
```
Comment (Zylphrex) [inline static/app/components/profiling/FlamegraphViewSelectMenu.tsx:31]:
```suggestion
        t('Left Heavy')
```

File: static/app/components/profiling/FlamegraphViewSelectMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      </Button>
+      <Button
+        size="xsmall"
+        priority={view === 'bottom up' ? 'primary' : undefined}
+        onClick={() => onViewChange('bottom up')}
+      >
+        Bottom Up
```
Comment (Zylphrex) [inline static/app/components/profiling/FlamegraphViewSelectMenu.tsx:38]:
```suggestion
        t('Bottom Up')
```

**Review by Zylphrex — APPROVED** [review]

File: static/app/components/profiling/FlamegraphViewSelectMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+function FlamegraphViewSelectMenu({
+  view,
+  onViewChange,
+  sorting,
+  onSortingChange,
+}: FlamegraphViewSelectMenuProps): React.ReactElement {
+  return (
+    <ButtonBar merged>
```
Comment (evanpurkhiser) [inline static/app/components/profiling/FlamegraphViewSelectMenu.tsx:19]: Is the gif outdated? Doesn't look like it's using the button bar

Comment (JonasBa) [reply static/app/components/profiling/FlamegraphViewSelectMenu.tsx:19]: Yes, indeed, it's outdated!

**Pull request merged**